frontend-react/src/components/SideNav.test.tsx

```diff
@@ -0,0 +1,118 @@
+import { screen } from "@testing-library/react";
+
+import { renderApp } from "../utils/CustomRenderUtils";
+
+import { SideNavItem } from "./SideNav";
+import { USNavLink } from "./USLink";
+
+// link -> li -> sidenav (ul) -> div
+function getIsVisible(e: HTMLElement) {
+    return !e.parentElement?.parentElement?.parentElement?.classList.contains(
```

Comment:
This feels fragile since it's so tightly coupled to an exact element hierarchy.  I'd recommend one of the following:

- Omit the entire [subnav](https://github.com/CDCgov/prime-reportstream/pull/8954/files#diff-0b6f9b990aac3ec98d4b3517114cacc1b3fff66b869546a5b049046ad5947be0R50-R54) OR add an `aria-hidden`/`hidden` if `isSubnavVisible` is `true`.  This would result in the query returning nothing for this link so we could run an expectation on that instead.
- Iterate upward through `parentElement`s until we find an ancestor element with a `display-none` class (since that should affect all descendants).  The other option is more preferable in my opinion since it's less reliant on a modifier class, though.